v1/pkg/client/model_volume.go

```diff
@@ -55,4 +55,6 @@ type Volume struct {
 	ActiveSite string `json:"ActiveSite"`
 	// The site where the volume was originally created.      
 	CreatedSite string `json:"CreatedSite"`
+	// The number of active exports for this volume
+	ExportCount int32 `json:"ExportCount"`
```

Comment:
Is there any reason not consistent with naming convention as in portal API model?
https://github.com/hpe-hcss/quake/blob/dd3d55c5afc9a4a7bb76f56daded9e0c3dd135cc/model/volume.go#L170

In portal api models, field name is like this:
json:"export_cnt"

Reply:
Yes, but as this will be exposed to end-users.
I though it is better not to name the variable in short form.
let me know if you want me to change it from `ExportCount` -> `ExportCnt`

Reply:
Hello @harinathakotha, AFAIK the only convention we have for Project API is camel case. Both `ExportCount ` and `ExportCnt ` are fine.